packages/x-data-grid-premium/src/hooks/features/aggregation/createAggregationLookup.ts

```diff
@@ -140,7 +157,7 @@ export const createAggregationLookup = ({
     }
 
     const hasAggregableChildren = groupNode.children.length;
-    if (hasAggregableChildren) {
+    if (hasAggregableChildren || isDataSource) {
       const position = getAggregationPosition(groupNode);
       if (position != null) {
         aggregationLookup[groupNode.id] = getGroupAggregatedValue({
```

Comment:
```suggestion
        aggregationLookup[groupNode.id] = getGroupAggregatedValueWithDataSource({ // or some other name
          groupId: groupNode.id,
          apiRef,
          aggregatedFields 
        })
```
Since you don't need other parameters in that case and you don't have to propagate `isDataSource` twice